java/src/main/java/com/saucelabs/simplesauce/examples/UsingJUnit.java

```diff
@@ -0,0 +1,39 @@
+package com.saucelabs.simplesauce.examples;
+
+import com.saucelabs.simplesauce.SauceSession;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+
+public class UsingJUnit {
+
+    private SauceSession sauceSession;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'sauceSession' could be replaced by a local variable.](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4550797)